src/prefect/tasks/gcp/storage.py

```diff
@@ -22,7 +22,7 @@ def __init__(
         chunk_size: int = 104857600,  # 1024 * 1024 B * 100 = 100 MB
         encryption_key_secret: str = None,
         request_timeout: Union[float, Tuple[float, float]] = 60,
-        **kwargs
+        **kwargs,
```

Comment:
Weird behavior from `black` here, should I keep it that way or remove the change ?

Reply:
A trailing comma is standard from `black` although this will break in py2.7 which doesn't matter since we require python3 -- so this change should be fine.